### PR TITLE
Reverse toggle

### PR DESCRIPTION
**Pull request by benrudolph**
@orangejenny thanks, definitely think we should do this

### CODE REVIEW

**Comment by orangejenny** [comment]
thanks for fixing test :)

**Comment by benrudolph** [comment]
✅

File: corehq/toggles.py [hunk 1]
```diff
@@ -1017,16 +1017,11 @@ def _commtrackify(domain_name, toggle_is_enabled):
...
-    'Use Nimbus to validate XForms',
+FORMTRANSLATE_FORM_VALIDATION = StaticToggle(
+    'formtranslate_form_validation',
+    'Use formtranslate to validate XForms',
     TAG_PRODUCT_PATH,
-    [NAMESPACE_DOMAIN],
-    randomness=1.0,
-    always_disabled=[
-        'icrc-almanach',
-        'mikolo'
```
Comment (czue) [inline corehq/toggles.py]: does this need to be coordinated with deployer to get added?

**Review by czue — APPROVED** [review]

File: corehq/toggles.py (hunk 1 above)
Comment (benrudolph) [inline corehq/toggles.py]: more important that the swiss one gets added when swiss is deployed. we added mikolo due to memory issues in web apps. i think that has now been fixed. thanks for reminding

**Comment by benrudolph** [comment]
@emord @kaapstorm if swiss gets deployed during your interrupt, can you add `icrc-almanach` to this toggle?

File: corehq/toggles.py (hunk 1 above)
Comment (emord) [inline corehq/toggles.py]: If there's more of these, maybe it would make sense to implement always_enabled arument?

Comment (benrudolph) [inline corehq/toggles.py]: yeah, i think the functionality that would be nice to have is something like "initial" where we seed the flag with a few domains, but we're still able to turn them off without a code change. `always_disabled` is useful when it's a random toggle

Comment (benrudolph) [inline corehq/toggles.py]: yeah, i think the functionality that would be nice to have is something like "initial" where we seed the flag with a few domains, but we're still able to turn them off without a code change. `always_disabled` is useful when it's a random toggle

**Comment by benrudolph** [comment]
handling the swiss deploy myself

**Pull request merged**